addons/mail/static/src/components/discuss_sidebar_mailbox/discuss_sidebar_mailbox.xml

```diff
@@ -1,23 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-name="mail.DiscussSidebarMailbox" owl="1">
-        <div class="o_DiscussSidebarMailbox"
+        <button class="o_DiscussSidebarMailbox btn d-flex align-items-center py-1 px-0 border-0 rounded-0 font-weight-normal text-reset"
```

Comment:
TBH I don't feel super confident with `<button>` having complex content.

For me it should be ideally as flexible as `<div>` and just built-in styling for buttons...
But I have an _a priori_ that it could show some strange and annoying behaviours.
I really hope it's not the case!

Reply:
I think it's fine as `<button>` for now, as it works and it's more functionally meaningful and brief while code is still "low-level" (by that, I mean it's pretty much to what's as lowly and natively available to the platform we target, the web, including lots of technical stuffs and poorly designed features).

I think eventually, we'd want buttons that are more complex and that just work on any targeted platform. So having something as flexible as a plain `<div/>` technically in web platform might be desirable.
Maybe when we'd need it, we'd just introduce a component `Button`... I don't know exactly when and how, but it's not today so let's go with `<button>` (lowercase) for now ;)